Envfile

```diff
@@ -164,4 +164,21 @@ variable :SMARTY_STREETS_WEB_KEY, :String, default: "Optional"
 
 group :production do
   variable :SECRET_KEY_BASE, :String
+  
+  # Timber
+  variable: :SEND_LOGS_TO_TIMBER, :String  # Defaults to true in code
```

Comment:
It looks like this, and those below should be `variable` without the colon at the end.

Reply:
Whoops!  You're totally right.  Fixed!